libs/community/langchain_community/vectorstores/pathway.py

```diff
@@ -0,0 +1,141 @@
+"""
+Pathway vector store server and client.
+
+
+The PathwayVectorServer builds a pipeline which indexes all files in a given folder,
+embeds them and builds a vector index. The pipeline reacts to changes in source files,
+automatically updating appropriate index entries.
+
+The PathwayVectorClient implements the LangChain VectorStore interface and queries the
+PathwayVectorServer to retrieve up-to-date documents.
+
+"""
+
+from typing import Callable, List, Optional
+
+import pathway as pw
```

Comment:
this should be a conditional import

Reply:
Sorry about the omission, all should be optional now.